plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveMetadata.java

```diff
@@ -2774,6 +2774,18 @@ public void revokeRoles(ConnectorSession session, Set<String> roles, Set<TrinoPr
         return accessControlMetadata.listEnabledRoles(session);
```

Comment:
The convention for commit messages is to use imperative mode, like "Add grant/revoke..." and to keep the headline relatively short (I know it can be difficult at times - I struggle with it myself ;) )

(And it's annoying how GitHub does not allow commenting on the commit message)